Source/Scene/PointCloud3DTileContent.js

```diff
@@ -1004,7 +1020,13 @@ define([
             vs += '    gl_PointSize = u_pointSize; \n';
         }
 
-        vs += '    color = color * u_highlightColor; \n';
+        vs += '    vec4 position_EC = czm_view * vec4(position_absolute, 1.0); \n' +
+              '    position_EC.z *= -1.0; \n\n' +
+              '    float attenuationFactor = \n' +
+              '    ((position_EC.z - czm_clampedFrustum.x) /\n' +
+              '    (czm_clampedFrustum.y - czm_clampedFrustum.x)); \n' +
+              '    gl_PointSize *= mix(u_pointAttenuationMaxSize, 1.0, attenuationFactor); \n\n' +
```

Comment:
The point attenuation code should be added to the shader only if `_pointAttenuationMaxSize > _pointSize`. Also point attenuation should probably not be done at all if the style has a `pointSize` property.